modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -30,7 +40,7 @@ object ObservationDao {
        .unique
        .map { case (t, i) =>
          Observation(id, t, i, Nil)
-       }      
+       }
 
   def select(id: Observation.Id): ConnectionIO[Observation[Step[_]]] =
```

Comment:
As long as it is possible to delete observations the possibility of holding an ID that no longer exists is there.  I suppose though the idea is to simply mark them deleted rather than actually delete them.